### PR TITLE
Add quit game functionality

### PR DESCRIPTION
**Pull request by adamgilroy22**

### CODE REVIEW

File: assets/js/scenes/gameScene.js [hunk 1]
```diff
@@ -252,6 +237,20 @@ const createGameScene = () => {
       layer("ui"),
       scale(0.4),
     ]);
+
+    // Quit Game
+    onKeyDown("escape", () => {
+      mainMusic.stop();
```
Comment (stephendawsondev) [inline assets/js/scenes/gameScene.js:243]: Good call on stopping the music

File: assets/js/scenes/gameScene.js [hunk 2]
```diff
@@ -252,6 +237,20 @@ const createGameScene = () => {
...
+      go("welcome");
+    });
+
+    // Display Quit Game Text
+    add([
+      text("Esc: Quit"),
+      pos(width() * 0.8, 0),
+      layer("ui"),
+      scale(0.4),
+    ]);
```
Comment (stephendawsondev) [inline assets/js/scenes/gameScene.js:253]: Issue (non-blocking): I can't seem to see this text - is it appearing for you?

File: assets/js/scenes/gameScene.js [hunk 3]
```diff
@@ -252,6 +237,20 @@ const createGameScene = () => {
       layer("ui"),
       scale(0.4),
     ]);
+
+    // Quit Game
+    onKeyDown("escape", () => {
+      mainMusic.stop();
+      go("welcome");
```
Comment (stephendawsondev) [inline assets/js/scenes/gameScene.js:244]: Screenshot 2023-08-20 at 09 58 55

**Review by stephendawsondev — CHANGES_REQUESTED** [review]
The quit works as expected, but it looks like the text is missing from the screen. Is that appearing for you? 

If not, you might just want to remove it.

File: assets/js/scenes/gameScene.js (hunk 3 above)
Comment (adamgilroy22) [reply assets/js/scenes/gameScene.js:244]: Yep! 
image

File: assets/js/scenes/gameScene.js (hunk 2 above)
Comment (adamgilroy22) [reply assets/js/scenes/gameScene.js:253]: image

File: assets/js/scenes/gameScene.js (hunk 3 above)
Comment (stephendawsondev) [reply assets/js/scenes/gameScene.js:244]: Not sure why that wasn't showing before, but it is now! Thanks!

**Review by stephendawsondev — APPROVED** [review]
LGTM! 🚢

**Pull request merged**